src/style.css

```diff
@@ -0,0 +1,180 @@
+@font-face {
+    font-family: Harry;
+    src: url("images/Parry\ Hotter.ttf");
+  }
+
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    font-family: "Harry";
+    color:rgb(245, 248, 71);
+}
+html{
+    font-size:0.7vw;
+}
+a{
+    color:#EFF24E;
+    text-decoration:none;
+    font-size:2rem;
+}
+body {
+    background-image: url("images/f9d7af29e8b87e7bdbc81609503036d1.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: bottom;
+}
+main{
+    position:relative;
+    height:100vh;
+}
+@media only screen and (min-width: 600px) {
+    main{
+    height:190vh;
+}
+}
+.main_feather, .main_book_shadow, .h_hogwart{
+    width:7vw;
+}
+
+nav{
```

Comment:
Podczas przesuwania strony w dół tekst z nav najeżdża na treść strony i uniemożliwia wybranie domu np. Gryfindoru. Może lepiej byłoby zrobić to tak, żeby ten pasek miał stałe tło albo po prostu zasłaniał sobą treść strony pod spodem? Bo czasem strona robi się nieczytelna.